ext/build_defaults.yaml

```diff
@@ -1,4 +1,5 @@
 ---
 packaging_url: 'git://github.com/puppetlabs/packaging.git --branch=master'
 packaging_repo: 'packaging'
+project: 'puppetdb'
 build_tar: FALSE
```

Comment:
should this line also be removed?

Reply:
@underscorgan yes, but I want to get my packaging fix in first.

Reply:
which only exists in my brain atm